tests/unittests/threed/object_3d_tests.py

```
@@ -314,3 +314,467 @@ def test_traverse(self):
+        """Test removing by UUID, then by name."""
+        self.root.remove_by_uuid(self.child2.uuid)
+        self.assertFalse(any(c is self.child2 for c in self.root.children))
+
+        self.root.remove_by_name("grandchild3")
+        self.assertFalse(
+            any(c is self.grandchild3 for c in self.child3.children)
+        )
+        self.assertEqual(len(self.child3.children), 0)
+
+
+class TestObject3DFindAndExecute(unittest.TestCase):
```

Pylint (4.0.3)

[convention] Missing class docstring

(C0115)